microprofile/tests/tck/tck-lra/pom.xml

```diff
@@ -108,10 +108,6 @@
                     <includes>
                         <include>**/*Test*.java</include>
                     </includes>
-                    <excludes>
-                        <!-- It fails randomly -->
-                        <exclude>org.eclipse.microprofile.lra.tck.TckRecoveryTests.java</exclude>
-                    </excludes>
```

Comment:
:eyes: This is LRA not a Fault Tolerance and it was fixed https://github.com/helidon-io/helidon/commit/8a041bc063ff39f9b75a665f89663021f5eb2c76 a week ago, why it was excluded in the first place?

Reply:
I have updated it the PR and the original issue. It is LRA.

It was excluded because test was failing very often locally and when it was executed by the github actions. I don't know what has changed but now it is consistently working.